mqtt/mqtt-edgehub/tests/command.rs

```diff
@@ -1,19 +1,16 @@
-use std::collections::HashMap;
+use std::error::Error as StdError;
```

Comment:
Do we have tests for Authorized Identities commands?

Reply:
I don't see any. @dylanbronson may know better